pmm-app/src/pmm-add-instance/components/AddRemoteInstance/FormParts/ExternalServiceConnectionDetails/ExternalServiceConnectionDetails.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { FC, useMemo } from 'react';
+import { PasswordInputField, TextInputField, validators } from '@percona/platform-core';
+import Validators from 'shared/components/helpers/validators';
+import { LinkTooltip } from 'shared/components/Elements/LinkTooltip/LinkTooltip';
+import { Button, useTheme } from '@grafana/ui';
+import { Field } from 'react-final-form';
+import { RadioButtonGroupAdapter } from 'shared/components/Form/FieldAdapters/FieldAdapters';
+import { ExternalServiceConnectionDetailsFormPartProps } from '../FormParts.types';
+import { getStyles } from '../FormParts.styles';
+import { Messages } from '../FormParts.messages';
+import { schemaOptions } from '../FormParts.constants';
+
+export const ExternalServiceConnectionDetails: FC<ExternalServiceConnectionDetailsFormPartProps> = ({
+  form,
+}) => {
+  const theme = useTheme();
+  const styles = getStyles(theme);
+
+  const portValidators = useMemo(() => [validators.required, Validators.validatePort], []);
+
+  const getUrlParts = (value) => {
+    try {
+      const url = new URL(form.getState().values.url);
+      const protocol = url.protocol.replace(':', '');
+
+      form.mutators.setValue('schema', protocol);
+      form.mutators.setValue('address', url.hostname);
+      form.mutators.setValue('port', url.port || (protocol === 'https' ? '443' : '80'));
+      form.mutators.setValue('metrics_path', url.pathname);
+      form.mutators.setValue('username', url.username);
+      form.mutators.setValue('password', url.password);
+    } catch (e) {
+      console.log(e);
+    }
+
+    return value;
+  };
+
+  return (
+    <div className={styles.groupWrapper}>
+      <h4 className={styles.sectionHeader}>External service connection details</h4>
```

Comment:
Some strings need to go to Messages file.

Reply:
done, thanks